doc/source/tutorial/stats/hypothesis_chisquare.md

```diff
@@ -0,0 +1,67 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+```{eval-rst}
+.. jupyterlite:: ../../_contents/hypothesis_chisquare.ipynb
+   :new_tab: False
+```
+
+(hypothesis_chisquare)=
+# Hypothesis testing: Chi-square test
```

Comment:
If the other examples going here follow the precedent, all will begin "Hypothesis testing". Is that intentional?

Reply:
I was of two minds about it so we can remove if we feel this is not needed, especially if these documents all live under a toc with "Hypothesis testing" caption/title or something similar.

Reply:
I think they will.